ADVANCED_README.md

```diff
@@ -1298,8 +1298,8 @@ Every data, states, websocket session, session data, must be shared via any kind
 
 We recommend using Redis for sharing session data, websocket.
 
-- SocketIO with Redis : [https://github.com/LearnBoost/Socket.IO/wiki/Configuring-Socket.IO](Configuring SocketIO)
-- Redis session store for Connect : [https://github.com/visionmedia/connect-redis](Connect-redis)
+- SocketIO with Redis : [https://github.com/LearnBoost/Socket.IO/wiki/Configuring-Socket.IO](https://github.com/LearnBoost/Socket.IO/wiki/Configuring-Socket.IO)
```

Comment:
Thanks for the PR, do you mind changing theses links so that a label appear instead of the raw link?
```
[Configuring SocketIO](https://github.com/LearnBoost/Socket.IO/wiki/Configuring-Socket.IO)
```

Same goes for the link below,

Cheers,